src/__tests__/pages/ussf-documentation.test.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { screen, waitFor, act } from '@testing-library/react'
+import type { RenderResult } from '@testing-library/react'
+import { MockedProvider } from '@apollo/client/testing'
+import { useRouter } from 'next/router'
+import { axe } from 'jest-axe'
+import axios from 'axios'
+import { renderWithAuth } from '../../testHelpers'
+import USSFDocumentation from 'pages/ussf-documentation'
+
+const mockReplace = jest.fn()
+
+jest.mock('next/router', () => ({
+  useRouter: jest.fn(),
+}))
+
+jest.mock('axios')
+
+const mockedUseRouter = useRouter as jest.Mock
+
+mockedUseRouter.mockReturnValue({
+  route: '',
+  pathname: '',
+  query: '',
+  asPath: '',
+  push: jest.fn(),
+  replace: mockReplace,
+})
+
+describe('USSF Documentation page', () => {
+  describe('without a user', () => {
+    beforeEach(() => {
+      renderWithAuth(
+        <MockedProvider>
+          <USSFDocumentation />
+        </MockedProvider>,
+        { user: null }
+      )
+    })
+
+    it('renders the loader while fetching the user', () => {
+      expect(screen.getByText('Content is loading...')).toBeInTheDocument()
+    })
+
+    it('redirects to the login page if not logged in', async () => {
+      await waitFor(() => {
+        expect(mockReplace).toHaveBeenCalledWith('/login')
+      })
+    })
+  })
+
+  describe('when logged in', () => {
+    let html: RenderResult
+
+    beforeEach(() => {
+      html = renderWithAuth(
+        <MockedProvider>
+          <USSFDocumentation />
+        </MockedProvider>
+      )
+    })
```

Comment:
Thought, is there a reason to continue this pattern of sharing `html`. IIRC the recommendation is to not do that now with react-testing-library. I think this was the article: https://kentcdodds.com/blog/avoid-nesting-when-youre-testing

Suggestion: refactor this to call render in each it block or just combine them all into one test. I can't find the article that recommended that but I read one from Kent Dodds about that.